leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/HeapAnalysisListScreen.kt

```diff
@@ -62,7 +57,7 @@ internal class HeapAnalysisListScreen : Screen() {
         SimpleListAdapter(R.layout.leak_canary_leak_row, projections) { view, position ->
           val titleView = view.findViewById<TextView>(R.id.leak_canary_row_text)
           val timeView = view.findViewById<TextView>(R.id.leak_canary_row_time)
-          val index = count - position
+          val index = position + 1
```

Comment:
@pyricau changed this so items in the list show up in order. Did you want them showing in reverse though? This seemed pretty intentional.

Reply:
The results are sorted by timestamp DESC so the top item of the list is the last occurence, hence the reversed index.

Honestly the whole UX needs to be redesigned to figure out what matters or not and what information to surface.